packages/rspeedy/plugin-react/src/entry.ts

```diff
@@ -137,7 +138,7 @@ export function applyEntry(
         })
         // in standalone lazy bundle mode, we do not add
         // other entries to avoid wrongly exporting from other entries
-        .when(isDev && !isWeb, entry => {
+        .when(enabledHMR, entry => {
```

Comment:
This should be different with the previous condition.

See comment at https://github.com/lynx-family/lynx-stack/pull/1885#discussion_r2426469734